core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -216,6 +216,16 @@ long getNewBlockIdForFile(AlluxioURI path) throws FileDoesNotExistException, Inv
    */
   Map<String, MountPointInfo>  getMountTable();
 
+  /**
+   * Translates given URI to an Alluxio URI.
+   * If given URI does not contain a scheme, then it is regarded as Alluxio URI.
+   *
+   * @param uriStr uri
+   * @return URI in Alluxio namespace
+   * @throws InvalidPathException
+   */
+  AlluxioURI translateUri(String uriStr) throws InvalidPathException;
```

Comment:
nit: `translateUri` is not very clear what are we translating the URI to. How about `reverseLookup` as you commented below?

Reply:
There is actually a translation here because it supports both Alluxio and non-Alluxio URIs. Updated the comment for clarity. 

MountTable#translate on the other hand doesn't do translation but do strict reverse lookup. Changed method name there.